targets/alert-cli/src/index.ts

```diff
@@ -219,14 +221,14 @@ async function getAlertChangesFromTags(
   const diff = await currTree.diff(prevTree);
   const patches = await diff.patches();
 
-  return diffProcessor(
-    repository,
-    currentTag,
-    patches,
-    fileFilter,
-    prevTree,
-    currTree
-  );
+  return diffProcessor({
+    currTree: currTree,
+    fileFilter: fileFilter,
+    patches: patches,
+    prevTree: prevTree,
+    repositoryId: repository,
+    tag: currentTag,
```

Comment:
tu aurais pu faire cela

```suggestion
return diffProcessor({
    currTree,
    fileFilter,
    patches,
    prevTree,
    repositoryId: repository,
    tag: currentTag,
```

Reply:
J'avoue, je n'ai que copier/coller les fonctions sans retravailler le contenu 😄